examples/basic/packages/ui/package.json

```diff
@@ -1,21 +1,36 @@
 {
-  "name": "ui",
+  "name": "@repo/ui",
   "version": "0.0.0",
-  "main": "./index.tsx",
-  "types": "./index.tsx",
+  "private": true,
   "license": "MIT",
+  "exports": {
+    "./card": {
+      "types": "./src/card.tsx",
+      "development": "./src/card.tsx",
+      "default": "./dist/card.js"
+    },
+    "./code": {
+      "types": "./src/code.tsx",
+      "development": "./src/code.tsx",
+      "default": "./dist/code.js"
+    }
+  },
   "scripts": {
-    "lint": "eslint .",
+    "build": "tsc",
+    "lint": "eslint . --max-warnings 0",
+    "typecheck": "tsc --noEmit",
     "generate:component": "turbo gen react-component"
   },
   "devDependencies": {
+    "@repo/eslint-config": "workspace:*",
+    "@repo/typescript-config": "workspace:*",
     "@turbo/gen": "^1.10.12",
     "@types/node": "^20.5.2",
+    "@types/eslint": "^8.44.7",
     "@types/react": "^18.2.0",
     "@types/react-dom": "^18.2.0",
-    "eslint-config-custom": "workspace:*",
+    "eslint": "^8.53.0",
```

Comment:
I'm split about this one. Keeping eslint at root will save you a lot of headache (like we do for prettier) because it guarantees there's only one copy, with the same version, and every workspace resolves the same one. Specifing it per workspace also works, but can become a bigger maintenance headache if things start to drift (time for syncpack?!)

Reply:
Oops, no, this is fluff. Delete!